src/tunacode/core/agents/resume/summary.py

```
@@ -16,25 +16,30 @@
 
-# Summary generation threshold (tokens)
-SUMMARY_THRESHOLD: int = 40_000
-
 # Summary marker for detection
 SUMMARY_MARKER: str = "[CONVERSATION_SUMMARY]"
 
+
+class SummaryGenerationError(Exception):
+    """Raised when summary generation fails."""
+
+    pass
```

Ruff (0.14.14)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)